src/app/app.module.ts

```diff
@@ -5,7 +5,8 @@ import { HttpClient, HttpClientModule } from '@angular/common/http'
 import { TranslateModule, TranslateLoader } from '@ngx-translate/core'
 import { TranslateHttpLoader } from '@ngx-translate/http-loader'
 
-import { AppComponent } from './app.component'
+import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
```

Comment:
could you move the import to the second block of imports, 3rd party libs please

Reply:
explain

Reply:
what we have talked yesterday, when I show you 3 blocks of imports